infra/fastly/terraform/dl.k8s.io/services.tf

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+locals {
+  domain_name = "cdn.dl.k8s.io"
+}
+
+resource "fastly_service_vcl" "files" {
+  name     = local.domain_name
+  activate = true
+
+  domain {
+    name = local.domain_name
+  }
+
+  backend {
+    name             = "GCS"
+    auto_loadbalance = false
+
+    healthcheck = "GCS Health"
+
+    address           = "storage.googleapis.com"
+    port              = 443
+    use_ssl           = true
+    ssl_cert_hostname = "storage.googleapis.com"
+    ssl_sni_hostname  = "storage.googleapis.com"
+
+    override_host = "${var.bucket}.storage.googleapis.com"
+
+    connect_timeout       = 5000  # milliseconds
+    first_byte_timeout    = 60000 # milliseconds
+    between_bytes_timeout = 15000 # milliseconds
+    error_threshold       = 5
+  }
+
+  healthcheck {
+    name = "GCS Health"
+
+    host           = "${var.bucket}.storage.googleapis.com"
+    method         = "GET"
+    path           = "/"
+    check_interval = 3000
+    timeout        = 2000
+    threshold      = 2
+    initial        = 2
+    window         = 4
+  }
+
+  product_enablement {
+    origin_inspector = true
+  }
+
+  snippet {
+    content  = <<-EOT
+      if (req.url.path ~ "^/release/") {
```

Comment:
Could move `/release` into a local or an input. Very much optional.

Reply:
From a code perspective, it make sense but this path is hardecoded in too many places to make variable. I prefer we make it explicit for the moment.